WordPress/WordPressTest/BloggingPromptsServiceTests.swift

```diff
@@ -196,29 +331,55 @@ private extension BloggingPromptsServiceTests {
             return fixture(filePath: stubPath!, headers: ["Content-Type": "application/json"])
         }
     }
+
+    @discardableResult
+    func makeBloggingPrompt(siteID: Int32, promptID: Int32? = nil, date: Date) -> BloggingPrompt {
+        let prompt = BloggingPrompt.newObject(in: contextManager.mainContext)!
+        prompt.siteID = siteID
+        prompt.date = date
+        prompt.promptID = promptID ?? NSNumber(value: arc4random_uniform(UInt32.max)).int32Value
```

Comment:
Worth noting that introducing randomness makes the tests non-deterministic. Is it worth not having to type different ids in the tests where we create more than one dummy prompt?

Reply:
Good point. I've addressed this in https://github.com/wordpress-mobile/WordPress-iOS/pull/21952/commits/841761ce677a705f17f4efd0d0ea3f026857107b.